packages/icons/icons.json

```
@@ -4156,5 +4156,33 @@
         "tags": "person, people, teammate, team, sports",
         "group": "miscellaneous",
         "codepoint": 62290
+    },
+    {
+        "displayName": "crop",
+        "iconName": "crop",
+        "tags": "crop, cut, resize",
```

no need to include icon name in the tags; the search logic incorporates both when querying for results

```suggestion
        "tags": "image, media, edit, cut, resize",
```